tests/WE2E/test_configs/wflow_features/config.specify_RESTART_INTERVAL.yaml

```diff
@@ -17,7 +17,8 @@ task_get_extrn_ics:
   USE_USER_STAGED_EXTRN_FILES: true
 task_get_extrn_lbcs:
   EXTRN_MDL_NAME_LBCS: FV3GFS
-  LBC_SPEC_INTVL_HRS: 3
+  LBC_SPEC_INTVL_HRS: 6
   USE_USER_STAGED_EXTRN_FILES: true
 task_run_fcst:
-  RESTART_INTERVAL: 1
+  WTIME_RUN_FCST: 01:00:00        
```

Comment:
Testing on Cheyenne showed that the `specify_RESTART_INTERVAL` test failed due to running out of walltime.  Please increase `WTIME_RUN_FCST:` for this job

Reply:
@MichaelLueken, I've removed this line, so its default value (4:30:00) will be used.